paperqa/agents/task.py

```diff
@@ -528,20 +536,16 @@ def extract_best_answer_index(self, text: str) -> int:
         match = re.search(r"<rating>(\d+)</rating>", text)
         return int(match.group(1)) if match else 0
 
-    async def pairwise_evaluation(
-        self, qid: str | UUID, question: str, pqa_answer: str, human_answer: str
-    ) -> dict:
-
+    async def _evaluate_answer(self) -> dict:
```

Comment:
@JoaquinPolonuer can you add a docstring to this after merging into your PR? Mention "pairwise evaluation" somehow